homeassistant/components/rabbitair/entity.py

```diff
@@ -0,0 +1,75 @@
+"""A base class for Rabbit Air entities."""
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import Any
+
+from rabbitair import Client, Model, State
```

Comment:
Client, Model & State are not re-exported from the package root.

```suggestion
from rabbitair.client import Client
from rabbitair.response import Model, State
```

Reply:
Fixed the library to properly re-export these imports.